shell/src/main/java/org/apache/accumulo/shell/commands/GetTabletHostingGoalCommand.java

```diff
@@ -48,11 +48,11 @@ public String description() {
 
   @Override
   protected void doTableOp(Shell shellState, String tableName) throws Exception {
-    List<HostingGoalForTablet> tabletHostingGoal = shellState.getAccumuloClient().tableOperations()
-        .getTabletHostingGoal(tableName, range).collect(Collectors.toList());
+    List<TabletInformation> tabletInformationList = shellState.getAccumuloClient().tableOperations()
+        .getTabletInformation(tableName, range).collect(Collectors.toList());
     shellState.getWriter().println("TABLE: " + tableName);
     shellState.getWriter().println("TABLET ID    HOSTING GOAL");
-    tabletHostingGoal.forEach(p -> shellState.getWriter()
+    tabletInformationList.forEach(p -> shellState.getWriter()
```

Comment:
May be able to do something like the following and avoid creating the list.

```suggestion
    shellState.getAccumuloClient().tableOperations()
        .getTabletInformation(tableName, range).forEach(p -> shellState.getWriter()
```